server/command.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mattermost/mattermost-plugin-zoom/server/zoom"
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/plugin"
+)
+
+const COMMAND_HELP = `* |/zoom start| - Start a zoom meeting.`
+
+func getCommand() *model.Command {
+	return &model.Command{
+		Trigger:          "zoom",
+		DisplayName:      "Zoom",
+		Description:      "Integration with Zoom.",
+		AutoComplete:     true,
+		AutoCompleteDesc: "Available commands: start",
+		AutoCompleteHint: "[command]",
+	}
+}
+
+func (p *Plugin) postCommandResponse(args *model.CommandArgs, text string) {
+	post := &model.Post{
+		UserId:    p.botUserID,
+		ChannelId: args.ChannelId,
+		Message:   text,
+	}
+	_ = p.API.SendEphemeralPost(args.UserId, post)
+}
+
+func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
```

Comment:
You are returning `&model.CommandResponse{}, nil` all the time. To make the code more clear, can you please wrap the code in a `executeCommand(c *plugin.Context, args *model.CommandArgs) (string, errror)` function and change `ExecuteCommand` in:

```go
func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
	msg, err := executeCommand(c,args)
	if err != mil {
		p.LogWarn("failed to execute command", "error", err.Error())
	}
	if msg != "" {
		p.postCommandResponse(args, msg)
	}
	return &model.CommandResponse{}, nil
```